docs/pages/user-manual.mdx

```diff
@@ -571,8 +571,8 @@ Host teleport.proxy
 ### Passing Teleport SSH certificate to OpenSSH client
 
 If a user does not want to use an SSH agent or if the agent is not available,
-the certificate must be passed to `ssh` via `IdentityFile` option (see `man
-ssh_config`). Consider this example: the Teleport user `joe` wants to login into
+the certificate must be passed to `ssh` via `IdentityFile` and `CertificateFile` options
+(see `man ssh_config`). Consider this example: the Teleport user `joe` wants to login into
 the proxy named `lab.example.com`.
```

Comment:
```suggestion
the Teleport cluster named `lab.example.com`.
```
The new naming scheme is `~/.tsh/keys/<proxy>/<username>-ssh/<cluster>-cert.pub`.

Reply:
Fixed in e8013922f289b8ff46566e162ed877ebccab68e0